common/src/proto.rs

```diff
@@ -114,3 +114,12 @@ pub struct Character {
 pub struct Inventory {
     pub contents: Vec<EntityId>,
 }
+
+pub mod connection_error_codes {
+    use quinn::VarInt;
+
+    pub const CONNECTION_LOST: VarInt = VarInt::from_u32(0);
+    pub const BAD_CLIENT_COMMAND: VarInt = VarInt::from_u32(2);
+    pub const STREAM_ERROR: VarInt = VarInt::from_u32(1);
+    pub const NAME_CONFLICT: VarInt = VarInt::from_u32(3);
```

Comment:
nit: numerical order

Reply:
Oops, good catch. I got the names mixed up and fixed them but forgot to reorder them.